sys/parsoid.js

```diff
@@ -430,7 +430,12 @@ PSP.getFormat = function(format, hyper, req) {
                     // Re-try to retrieve from storage with the
                     // normalized title & revision
                     rp.title = revInfo.title;
-                    return self.getFormat(format, hyper, req);
+                    return self.getFormat(format, hyper, req)
+                    .then(function(res) {
+                        // Disable caching for non-canonical requests, as
+                        // those are not purged.
+                        res.headers['cache-control'] = 'no-cache, must-revalidate';
```

Comment:
Why both `no-cache` **and** `must-revalidate` when the latter applies to a subset of the former?